src/assets/Logo.tsx

```diff
@@ -36,7 +39,7 @@ function LogoIcon({
         d="M37.0321 10.1313V10.029H35.572C34.885 10.029 34.3754 10.0972 34.0433 10.2337C33.7227 10.3587 33.5623 10.6374 33.5623 11.0695C33.5623 11.7177 34.0204 12.0418 34.9365 12.0418C35.7037 12.0418 36.2419 11.8826 36.5511 11.5642C36.8717 11.2344 37.0321 10.7568 37.0321 10.1313ZM34.3696 13.9182C33.2932 13.9182 32.4974 13.6567 31.9821 13.1335C31.4668 12.599 31.2091 11.911 31.2091 11.0695C31.2091 10.0574 31.5412 9.32958 32.2054 8.88606C32.8695 8.43118 33.883 8.20374 35.2457 8.20374H37.0321V7.82846C37.0321 6.98693 36.4939 6.56616 35.4174 6.56616C34.4555 6.56616 33.923 6.93007 33.82 7.65788H31.4152C31.4954 6.78223 31.8618 6.04873 32.5145 5.45738C33.1673 4.86603 34.1521 4.57036 35.469 4.57036C36.7744 4.57036 37.7592 4.86035 38.4234 5.44032C39.0876 6.0203 39.4196 6.83909 39.4196 7.89669V13.7135H37.2554V12.7241C36.7401 13.5202 35.7781 13.9182 34.3696 13.9182Z"
         fill={color}
       />
-      <rect y="20" width="40" height="3" fill={underlineColor} />
+      <rect y="20" width="40" height="3" fill={underlineColor || defaultUnderlineColor} />
```

Comment:
underlineColor (set as first) is always applied to the logo icon since you specify #FF414D color (the Alma red) in the Logo component:
`LogoIcon underlineColor="#FF414D" />`
![Capture d’écran du 2022-05-10 15-03-53](https://user-images.githubusercontent.com/104012464/167660793-08154d42-1382-4a4e-9a82-55cd846e7c78.png)